l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -227,7 +227,23 @@ def getCedPrest(self, cedPrest):
 
             if cedPrest.IscrizioneREA:
                 REA = cedPrest.IscrizioneREA
-                vals['rea_code'] = REA.NumeroREA
+                rea_partners = partner_model.search([
+                    ('rea_code', '=', REA.NumeroREA),
+                    ('id', '!=', partner_id)
+                ])
+                if rea_partners:
+                    rea_nr = REA.NumeroREA
+                    rea_names = ", ".join(rea_partners.mapped('name'))
+                    p_name = partner_model.browse(partner_id).name
+                    self.log_inconsistency(
+                        _("Current invoice is from {} with REA Code"
+                          " {}. Yet it seems that partners {} have the same"
+                          " REA Code. This code should be unique; please fix"
+                          " it before importing this invoice."
```

Comment:
LGTM. La segnalazione direi che cmq potrebbe essere senza `before importing this invoice` visto che a questo punto l'utente l'ha già importata.